src/package.json

```diff
@@ -0,0 +1,77 @@
+{
+    "name": "src",
+    "description": "React Native WebView component for iOS, Android, macOS, and Windows",
+    "main": "../index.js",
+    "typings": "../index.d.ts",
+    "author": "Jamon Holmgren <jamon@infinite.red>",
+    "contributors": [
+      "Thibault Malbranche <malbranche.thibault@gmail.com>"
+    ],
+    "license": "MIT",
+    "version": "11.3.2",
+    "private": true,
+    "homepage": "https://github.com/react-native-webview/react-native-webview#readme",
+    "scripts": {
+      "start": "node node_modules/react-native/local-cli/cli.js start",
+      "start:android": "react-native run-android",
+      "start:ios": "react-native run-ios",
+      "start:macos": "node node_modules/react-native-macos/local-cli/cli.js start --use-react-native-macos",
+      "start:windows": "react-native start --use-react-native-windows",
+      "ci": "CI=true && yarn lint",
+      "ci:publish": "yarn semantic-release",
+      "lint": "yarn tsc --noEmit && yarn eslint ./src --ext .ts,.tsx",
+      "build": "yarn tsc",
+      "prepare": "yarn build",
+      "appium": "appium",
+      "test:windows": "yarn jest --setupFiles=./jest-setups/jest.setup.windows.js"
```

Comment:
a lot of these are probably for the example app only (the starts, and I think the appium and tests)